TWLight/resources/filters.py

```diff
@@ -22,6 +23,13 @@ class PartnerFilter(django_filters.FilterSet):
         queryset=Language.objects.all(),
     )
 
+    searchable = django_filters.MultipleChoiceFilter(
+        # Translators: On the MyLibrary page (https://wikipedialibrary.wmflabs.org/users/my_library), this text is shown to indicate if a collection is searchable.
+        label=_("Searchable"),
+        choices=Partner.SEARCHABLE_CHOICES,
+        widget=forms.CheckboxSelectMultiple,
```

Comment:
I have tried to have the checkboxes checked by default here, but whenever I try to filter by searchability, the checkboxes stay checked. Maybe @jsnshrmn can help me figure this out.

Reply:
The easiest way to set this would be to just adjust the links/redirects to set the filter options in the url query parameters. e.g instead of sending people to:
`/users/my_library/`
send them to:
`/users/my_library/?searchable=0&searchable=1&searchable=2`

Reply:
I came up with another solution ...